pkg/network/deviceinfo/deviceinfo.go

```diff
@@ -49,7 +49,7 @@ func MapBindingPluginNetworkNameToDeviceInfo(networks []v1.Network,
 	)
 }
 
-func mapNetworkNameToDeviceInfo(networks []v1.Network,
+func MapNetworkNameToDeviceInfo(networks []v1.Network,
```

Comment:
Please add unit tests to this method.

Reply:
Done

The code is changed to filter device-info plugin and SR-IOV interfaces and use `MapNetworkNameToDeviceInfo`